receiver/prometheusreceiver/metrics_receiver.go

```diff
@@ -348,6 +348,8 @@ func (r *pReceiver) initAPIServer(ctx context.Context, host component.Host) erro
 		o.ConvertOTLPDelta,
 		o.NativeOTLPDeltaIngestion,
 		o.CTZeroIngestionEnabled,
+		time.Duration(5*time.Minute), // LookbackDelta - Using the default value of 5 minutes
```

Comment:
this is new? I assume this is for a different change being imported

Reply:
It's because of this PR: https://github.com/prometheus/prometheus/pull/16737